todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField()
```

Comment:
add _blank=True, default=''_ as arguments to make deadline optional, as it's declared in the task